src/main/scala/org/renci/cam/domain/package.scala

```diff
@@ -2,14 +2,28 @@ package org.renci.cam
 
 package object domain {
 
-  case class KGSQueryRequestBody(message: KGSMessage)
+  case class TranslatorQueryNode(id: String, `type`: String, curie: Option[String])
 
-  case class KGSMessage(query_graph: KGSQueryGraph)
+  case class TranslatorQueryEdge(id: String, `type`: String, source_id: String, target_id: String)
 
-  case class KGSQueryGraph(nodes: List[KGSNode], edges: List[KGSEdge])
+  case class TranslatorQueryGraph(nodes: List[TranslatorQueryNode], edges: List[TranslatorQueryEdge])
 
-  case class KGSNode(id: String, `type`: String, curie: Option[String])
+  case class TranslatorNodeAttribute(name: String, value: String, `type`: String, url: Option[String], source: Option[String])
 
-  case class KGSEdge(id: String, source_id: String, target_id: String, `type`: String)
+  case class TranslatorNode(id: String, name: Option[String], `type`: Option[List[String]], attributes: Option[List[TranslatorNodeAttribute]])
```

Comment:
Do you think we need to nest these `List`s in `Option`? Or just have an empty list if none?

Reply:
Unexpectedly, removing the Option solves the stack overflow problem. Can we do that?

Reply:
Some info here: https://circe.github.io/circe/codecs/known-issues.html

Reply:
I was assuming that anything not marked required in the api should be wrapped in an Option. For lists, I think that assumption is wrong & just having empty lists makes sense.

Reply:
Nice find on the known-issues.  I was having a terrible time finding the root cause as Scala, SBT, & Intellij are relatively new to me.